packages/connect-plugin-stellar/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@trezor/connect-plugin-stellar",
+    "version": "9.0.0",
+    "author": "Trezor <info@trezor.io>",
+    "homepage": "https://github.com/trezor/trezor-suite/packages/connect-plugin-stellar",
+    "description": "@trezor/connect plugin for Stellar",
+    "license": "SEE LICENSE IN LICENSE.md",
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/trezor/trezor-suite.git"
+    },
+    "bugs": {
+        "url": "https://github.com/trezor/trezor-suite/issues"
+    },
+    "keywords": [
+        "Trezor",
+        "trezor-connect",
+        "stellar"
+    ],
+    "sideEffects": false,
+    "main": "index.js",
+    "peerDependencies": {
```

Comment:
just wondering, should it be peerDependency, or regular dependecy?
what is the profit of making it peer?

Reply:
but theoretically it should be compatible with connect 8 as well, no?